src/main/webapp/css/style.css

```diff
@@ -1,3 +1,300 @@
 .favourite {
   color: yellow;
 }
+
+.jumbotron {
+    text-align: center;
+    background-color:#039BE5 !important;
+}
+
+/*.card {
+    text-align: center;
+    background-color:#2C384A !important;
+    width: 300px; 
+    height: 300px;
+    box-shadow: 5px 5px 5px 5px rgba(0, 0, 0, 0.43);
+   
+}*/
+
+.sdk {
+    text-align: center;
+    background-color:#2C384A !important;
+    width: 500px; 
+    height: 75px;
+    box-shadow: 5px 5px 5px 5px rgba(0, 0, 0, 0.15); 
+    color: white;
+    position: relative;
+    padding: 20px;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+
```

Comment:
Please keep the spacing consistent throughout the file. Check the style guide and add either one or two blank lines between all the code blocks.